service/matching/forwarder.go

```diff
@@ -200,6 +202,7 @@ func (fwdr *Forwarder) ForwardPoll(ctx context.Context) (*InternalTask, error) {
 
 	pollerID, _ := ctx.Value(pollerIDKey).(string)
 	identity, _ := ctx.Value(identityKey).(string)
+	isolationGroup, _ := ctx.Value(_isolationGroupKey).(string)
```

Comment:
This context.Value() is... probably nil safe, but mgiht be worth double-checking